File: sub-model/src/main/java/org/github/base/service/impl/TabSafeQuestionDataServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.TabSafeQuestionDataEntity;
import org.github.base.mapper.ITabSafeQuestionDataMapper;
import org.github.base.service.ITabSafeQuestionDataService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 安全问题库 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-04-25
 */
@Service
public class TabSafeQuestionDataServiceImpl extends ServiceImpl<ITabSafeQuestionDataMapper, TabSafeQuestionDataEntity> implements ITabSafeQuestionDataService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/TabSafeQuestionDataServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.TabSafeQuestionDataEntity;
import org.github.mysql.mydb.base.mapper.ITabSafeQuestionDataMapper;
import org.github.mysql.mydb.base.service.ITabSafeQuestionDataService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 安全问题数据项表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class TabSafeQuestionDataServiceImpl : ServiceImpl<ITabSafeQuestionDataMapper, TabSafeQuestionDataEntity>(), ITabSafeQuestionDataService
